src/components/skill-pill-component/SkillPillComponent.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Skill } from '../../models/skill';
+
+interface ISkillPillProps {
+    
+}
+
+export class SkillPillComponent extends React.Component<ISkillPillProps, any> {
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            allSkills: []
+        };
+    }
+
+    render() {
+        return (
+            <ul>
+                {this.state.allSkills.map((s: Skill) => {
+                    return <div>
+                        <p>{s.name}</p>
+                    </div>
+                })}
+            </ul>
+        )
+    }
+}
```

Comment:
Instead of having one component that renders every single skill, why not make a component that renders a single skill. Or maybe have both.

Reply:
Alec, this is a component that renders a single skill. We are planning to use the array mapping feature in a larger component to loop through the arrays. Considering putting color mapping logic on this component. We can have both methods of rendering a skill on the next pull request

Reply:
--Sam